generators/pythonv2/codegen/src/ast/core/Writer.ts

```diff
@@ -8,36 +8,17 @@ export declare namespace Writer {}
 
 export class Writer extends AbstractWriter {
     /* Import statements */
-    private references: Record<ModulePath, Reference[]> = {};
+    private references: Reference[] = [];
```

Comment:
The writer becomes way more simple as a result of all this. Notably it's still responsible for tracking references, as discussed above.